tt_metal/impl/dispatch/kernel_config/dispatch.cpp

```diff
@@ -199,9 +198,8 @@ void DispatchKernel::GenerateDependentConfigs() {
             dependent_config_.downstream_s_logical_core = UNUSED_LOGICAL_CORE;
         }
         dependent_config_.downstream_logical_core = UNUSED_LOGICAL_CORE;
-        dependent_config_.downstream_cb_base = my_dispatch_constants.dispatch_buffer_base();
-        dependent_config_.downstream_cb_size =
-            (1 << dispatch_constants::DISPATCH_BUFFER_LOG_PAGE_SIZE) * my_dispatch_constants.dispatch_buffer_pages();
+        dependent_config_.downstream_cb_base = 0;
```

Comment:
Are these unused as well?

Reply:
Yup. These are unused, looking at the old code in device.cpp, for HD and H it's not used. But for D it's used. CI is still in progress

Reply:
CI passed now
https://github.com/tenstorrent/tt-metal/actions/runs/12507556070

Reply:
Sounds good, let's comment theses as unused here as well, just so it's clear